compliance_checker/__init__.py

```diff
@@ -1 +1,2 @@
 __version__="1.1.1"
+__stdname_table__="v29"
```

Comment:
This should either have the string `cf` involved in the name, or moved to a cf specific file instead of the package init.